### ISSUE TITLE
Improve Accessibility of Components - Styling

### ISSUE THREAD

**Issue by livoszlak** [livoszlak/book-finder-api-2.0#4]
Review and improve the accessibility of the components, ensuring they meet WCAG guidelines.

This issue's scope should focus on styling/CSS as it pertains to the components. This includes color, size of elements, margins, button sizes, et c. According to [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) the background color with the nearly-black font color combo works - let's make sure the book cards and everything else also works to the WCAG guidelines. Color changes are welcome, as long as the colors look nice together and meet the guidelines.

**Comment by om9090** [comment]
Hi @livoszlak I would love to make UI changes and try different colors to make the website more appealing and interactive. Can you please assign the issue to me?

**Comment by livoszlak** [comment]
@om9090 Great! Thank you! 🙌

**Comment by om9090** [comment]
Yes thank you I’ll start working on it.
